transport/client.go

```diff
@@ -311,7 +315,7 @@ func Dial(address string, tr *Transport) (*tls.Conn, error) {
 
 	for _, chain := range state.VerifiedChains {
 		for _, cert := range chain {
-			revoked, ok := revoke.VerifyCertificate(cert)
```

Comment:
one example, flag.Int is actually "flag.CommandLine.Int"
https://golang.org/src/flag/flag.go?s=17993:18044#L584
All methods are declared on FlagSets, and CommandLine is the default FlagSet.